Roomblr/ParseClient.swift

```diff
@@ -114,21 +114,6 @@ class ParseClient {
         }
     }
     
-    // return all PFUsers from Parse
-    func fetchAllPFUseres(completion: (pfUsers: [PFUser]?, error: NSError?) -> ()) {
-        var query = PFUser.query()
-        
-        query!.findObjectsInBackgroundWithBlock { (objects: [PFObject]?, error: NSError?) -> Void in
-            if error == nil {
-                // assume [PFObject] are PFUsers
-                var pfUsers = objects as! [PFUser]
-                completion(pfUsers: pfUsers, error: nil)
-            } else {
-                print("ERROR: Could not query for PFUsers")
-                completion(pfUsers: nil, error: error)
-            }
-        }
-    }
```

Comment:
why is this function deleted? it's still used inside function `getTopTagsFromPosts`?